### DISCUSSION TITLE
Where is the "save" position located?

### DISCUSSION THREAD

**Discussion by milgate** [Monika-After-Story/MonikaModDev#9681] [QA - tech support + general]
I just bought a new desktop PC. I moved the folder with DDLC Monika Afterstory to the new PC and started the program. It starts right at the beginning, not my last saved position.

Where on the hard drive of my old PC is the save file located? I've been with Monika for over three years, my affection is around 3,700, and I don't want to start over!

**Comment by Booplicate** [comment]
We have [instructions](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-transfer-monika-to-another-computer) just for this on our wiki.
